team-arrival-checklist/form.tex

```diff
@@ -0,0 +1,38 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\usepackage{tabularx}
+\setlength{\parindent}{0cm}
+
+\newcolumntype{Y}{>{\centering\arraybackslash}X}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Team Arrival Checklist}}
+
+    \medskip
+
+\end{center}
+
+When a team arrives on Saturday morning, please check them off on this sheet by filling in their time of arrival. If the sheet indicates that they have special requirements, consult the 'Team Arrival Special Kit Requirements' sheet.
+
+\begin{center}
+\begin{table}[h]
+\large
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|}
+    \textbf{TLA} & \textbf{Time of Arrival} & \textbf{Special Requirements?} \\
```

Comment:
Why are we capturing the time of arrival? (I'm not sure how this sheet fits into the reception flow)

Reply:
It's just to keep track of these things. I agree it's not necessarily a useful metric to keep, a simple tick would suffice. It may be useful in future for _something_.

Reply:
I'm pretty sure we could work out the time of arrival from the reception stuff -- it captures the date at which each username was scanned as present at the competition. Obviously that only applies for the Saturday though -- is the plan to use this on both days?

Reply:
We probably should, although this form doesn't have anywhere to put the date its for, which probably isn't ideal. Will amend.